extensions/amp-audio/validator-amp-audio.protoascii

```diff
@@ -52,6 +52,10 @@ tags: {  # <amp-audio> for AMP (autoplay attribute allowed)
     name: "autoplay"
     value: ""
   }
+  attrs: {
```

Comment:
let's also add the new attribute to the validator tests https://github.com/ampproject/amphtml/blob/master/extensions/amp-audio/0.1/test/validator-amp-audio.html and https://github.com/ampproject/amphtml/blob/master/extensions/amp-audio/0.1/test/validator-amp-audio.out.

These tests essentially ensure the validation rule is correct a `preload` stays a valid attribute.

Reply:
Change made in latest commit.